envi/archs/ppc/__init__.py

```diff
@@ -151,7 +151,8 @@ def getPointerSize(self):
         return self.psize
 
     def pointerString(self, va):
-        return '0x%.8x' % va
+        # 8 byte addresses on a 64-bit platform
+        return '0x%.16x' % va
```

Comment:
good thinking, but i'm not sure this is necessary/beneficial.  if a 64-bit number is really only 48 significant bits, do we want to add huge numbers of 0's on the front?  what if it's only 24 significant bits?
i'm honestly asking.  what do you think?

Reply:
the `.` should prevent leading 0's, but I agree it probably wasn't necessary, I can change it back